### ISSUE TITLE
SEC_ERROR_BAD_SIGNATURE problems since last update

### ISSUE THREAD

**Issue by Khris777** [BrowserWorks/waterfox#366]
Since the latest update (56.0.2) I'm getting SEC_ERROR_BAD_SIGNATURE errors on multiple pages of the network of the company I'm working at.

Among these are my company's JIRA page as well as the HUE interface of one of our Hadoop clusters.

This only happens in Waterfox, Firefox and all other browsers can access all pages without issue.

**Comment by grahamperrin** [comment]
What's the user agent? You can tell this from a visit to <https://udger.com/resources/online-parser>

Please be prepared to test with a different agent, which can be set with an extension such as [Custom UserAgent String](https://addons.mozilla.org/addon/custom-useragent-string/), also known as _Custom User-Agent_.

**Comment by Khris777** [comment]
The userstring is

`Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:56.0) Gecko/20100101 Firefox/56.0.2 Waterfox/56.0.2`

**Comment by Squall-Leonhart** [comment]
fake a waterfox 57 user agent,  your company might have explicitly blocked 56 as a security risk.

**Comment by grahamperrin** [comment]
Also/alternatively please try 

`Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:56.0) Gecko/20100101 Firefox/56.0.2`

**Comment by Khris777** [comment]
Thank you for the suggestions, none of them worked sadly.

**Comment by Khris777** [comment]
For a moment I could not reach github and stackoverflow either. This happend after I had set the system date to tomorrow for a test, and back to today.

Afterwards both sites reported the error. I deleted all cookies and was able to access this page directly, afterwards I could login again and now github is working again. Stackoverflow also went back to working after I had deleted the cookies.

I don't know if that was expected behaviour.
